### PR TITLE
refactor: Rewrite the Frontend to match RedHat UI better

### PR DESCRIPTION
**Pull request by tumido**
Peek 2021-08-16 12-01

Homepage:
- Displays some additonal text and list of available Meteors
- It has RH footer
- It has top right tools poiniting to GitHub and Slack

Meteor details:
- Data refreshed every 5 seconds
- Status details points users directly to the OCP console
- Expiration time has a nice parsing/text to it
- Only the important details are shown

Not found page:
- Also rewritten, loosely follows console.redhat.com style

Under the hood:
- All server side rendered
- Client routing (no page reloads)
- Query OCP for console link

### CODE REVIEW

File: src/pages/index.jsx [hunk 1]
```diff
@@ -1,87 +1,60 @@
...
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate explicabo, distinctio eveniet quam voluptates rerum soluta assumenda
+          nihil laborum consequuntur eos magnam repellendus, maxime cum iusto illo? Maiores fugit autem ipsam ab ullam laborum, magni expedita
+          consectetur sint quas, enim mollitia libero hic! Eos, obcaecati? Sapiente corrupti pariatur et fuga.
+        </FlexItem>
+        <Divider isVertical />
+        <FlexItem style={{ maxWidth: '30%' }}>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates esse facilis a! Suscipit esse facere delectus quo pariatur autem
+          aliquid, quos dignissimos, ex fugit laborum. Dignissimos harum error ab quaerat atque non blanditiis cupiditate iusto minima sunt velit
+          praesentium illum deserunt obcaecati totam, dolores voluptatibus, porro earum voluptates odit ut? Sed molestiae amet, corrupti dolorem
+          laboriosam corporis adipisci aspernatur.
```
Comment (tumido) [inline src/pages/index.jsx:39]: @oindrillac @isabelizimm @margarethaley @durandom

can you help me with some text for this? I need some outline explaining and describing the Meteor project and it's integrations in here.

Comment (durandom) [reply src/pages/index.jsx:39]: 1. Use the same level of information as on https://mybinder.org/ - you can almost copy the text there, but with s/Docker/Container/ etc
2. Make sure to link back to this repo

Comment (isabelizimm) [reply src/pages/index.jsx:39]: Sure thing! I can put that in a different PR, if that's easier?

Comment (tumido) [reply src/pages/index.jsx:39]: @isabelizimm yes! let's do it that way! Lets merge this one then and you can open a follow up. :slightly_smiling_face:

**Review by isabelizimm — APPROVED** [review]
/lgtm

**Pull request merged**